### PR TITLE
Why we use Yarn

### PR DESCRIPTION
**Pull request by mskelton**
Adds a blog post about why we use Yarn at Widen.

### CODE REVIEW

**Comment by mskelton** [comment]
@dbeg @feltnerm Would appreciate a review on this when you have a chance.  Thanks!

File: _posts/2021-09-27-Why-We-Use-Yarn.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- `workspace:` - The workspace protocol creates a link to a package in another workspace which ensures that you will never accidentally download an old version of a workspace from the registry. This protocol is great for multi-package monorepos.
+
+## Miscellaneous niceties
+
+Yarn has plenty more nice features to talk about, including their flagship feature [Plug'n'Play](https://yarnpkg.com/features/pnp), but here are a few other small niceties that we enjoy using.
+
+- [Workspace constraints](https://yarnpkg.com/features/constraints) - Constraints are a very powerful way of ensuring consistency between packages in a monorepo. We use constraints extensively in our component and utility libraries.
+- `yarn up` wildcarding - The `yarn up` command supports wildcards to upgrade groups of dependencies (e.g., `yarn up '@babel/*'`).
+
+## Downsides of Yarn
```
Comment (feltnerm) [inline _posts/2021-09-27-Why-We-Use-Yarn.md:74]: Does Yarn still go through Facebook's servers?

**Review by feltnerm — APPROVED** [review]
lgtm 🧶 🐱

File: _posts/2021-09-27-Why-We-Use-Yarn.md (hunk 1 above)
Comment (mskelton) [reply _posts/2021-09-27-Why-We-Use-Yarn.md:74]: Nope. 

- https://yarnpkg.com/advanced/telemetry#will-my-information-go-to-facebook
- https://yarnpkg.com/getting-started/qa/#is-yarn-operated-by-facebook
- https://yarnpkg.com/getting-started/qa/#why-registryyarnpkgcom-does-facebook-track-us

File: _posts/2021-09-27-Why-We-Use-Yarn.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+title: Why We Use Yarn
+date: 2021-09-27
+author: Mark Skelton
+categories: yarn npm package-managers
+excerpt: "Choosing a JavaScript package manager is tough and there are lots of opinions on which one is best. Continue reading to learn why we chose Yarn as our package manager."
+---
+
+Much like operating systems, JavaScript package managers tend to be a source of much debate. One developer will swear by his package manager while another vows never to use it. Some developers simply don't care, and new developers often find themselves struggling to answer the question of which package manager to use.
```
Comment (dbeg) [inline _posts/2021-09-27-Why-We-Use-Yarn.md:9]: Thoughts on a gender neutral pronoun?

```suggestion
Much like operating systems, JavaScript package managers tend to be a source of much debate. One developer will swear by their package manager while another vows never to use it. Some developers simply don't care, and new developers often find themselves struggling to answer the question of which package manager to use.
```

File: _posts/2021-09-27-Why-We-Use-Yarn.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+Yarn has plenty more nice features to talk about, including their flagship feature [Plug'n'Play](https://yarnpkg.com/features/pnp), but here are a few other small niceties that we enjoy using.
+
+- [Workspace constraints](https://yarnpkg.com/features/constraints) - Constraints are a very powerful way of ensuring consistency between packages in a monorepo. We use constraints extensively in our component and utility libraries.
+- `yarn up` wildcarding - The `yarn up` command supports wildcards to upgrade groups of dependencies (e.g., `yarn up '@babel/*'`).
+
+## Downsides of Yarn
+
+While Yarn does have a lot of benefits like I just described, it comes with its share of downsides. I'm not going to cover these in great detail for the sake of time, but I wanted to at least mention them.
+
+- Non-standard - New users can't just run `yarn` in a fresh install of Node.
```
Comment (dbeg) [inline _posts/2021-09-27-Why-We-Use-Yarn.md:78]: Could a quick blurb be added about how to install it (perhaps noting that comically, the official yarn docs show installation via npm)?

File: _posts/2021-09-27-Why-We-Use-Yarn.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+---
+title: Why We Use Yarn
+date: 2021-09-27
+author: Mark Skelton
+categories: yarn npm package-managers
+excerpt: "Choosing a JavaScript package manager is tough and there are lots of opinions on which one is best. Continue reading to learn why we chose Yarn as our package manager."
```
Comment (dbeg) [inline _posts/2021-09-27-Why-We-Use-Yarn.md:6]: Tweak excerpt to match title? (or change title to use "chose"?)

```suggestion
excerpt: "Choosing a JavaScript package manager is tough and there are lots of opinions on which one is best. Continue reading to learn why we use Yarn as our package manager."
```

**Review by dbeg — CHANGES_REQUESTED** [review]

File: _posts/2021-09-27-Why-We-Use-Yarn.md (hunk 3 above)
Comment (mskelton) [reply _posts/2021-09-27-Why-We-Use-Yarn.md]: Good idea, updated

**Review by dbeg — APPROVED** [review]
💯 🚢 🧶

**Pull request merged**